src/nuget/xunit.console.netcore.nuspec

```diff
@@ -33,11 +33,21 @@
       <dependency id="System.Threading" version="4.0.10" />
       <dependency id="System.Threading.Tasks" version="4.0.10" />
       <dependency id="System.Xml.XDocument" version="4.0.10" />
+      <dependency id="xunit.runner.utility" version="2.1.0-rc1-build3168" />
     </dependencies>
   </metadata>
   <files>
     <!-- xunit runner for .NET Core -->
     <file src="xunit.console.netcore\xunit.console.netcore.exe" target="lib\aspnetcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dnxcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dotnet" />
+    
+    <!--
+      NuGet ignores .exe files in lib folders, so use .props files to copy the exe
+      to the output folder.
+    -->
+    <file src="xunit.console.netcore\xunit.console.netcore.exe" target="build\dotnet" />
+    <file src="xunit.console.netcore\xunit.console.netcore.props" target="build\dotnet" />
   </files>
```

Comment:
Are you guys using the testing targets we have in buildtools? If so it would be good to just get those updated to copy, if they don't already I cannot remember, instead.